### DISCUSSION TITLE
Possibly terminolgy changes

### DISCUSSION THREAD

**Discussion by MarceloHoffmeister** [community/community#23532] [New to GitHub]
<p>Hi,</p>
<p>Viewing my projects saw that section below:</p>
<p><img src="https://user-images.githubusercontent.com/23346589/181077586-711d9032-1385-4875-9b0d-1388fd88faf7.png" alt="image" data-base62-sha1="gtCEpDgxT1TmJufjb0lD2UuqDGC" width="310" height="125"></p>
<p>I don’t consider “Languages” the best terminology for this section. I understand GitHub takes “Languages” by file extension so, to avoid a lot of refactors, how about just change this section name to something like “Technologies” or “Languages and Technologies”?</p>
<p>ps: I don’t know how to submit code for GitHub site, don’t even know if that’s possible, so i made this issue my first discussion here.</p>

**Comment by Gisgar3** [comment, marked as answer]
<p>Hey there, welcome to the <strong>GitHub Support Community</strong>! I understand why you’d suggest updating the terminology for that section of repositories (as <strong>Vue</strong> is technically not a language, but a framework). I recommend suggesting feedback to GitHub through their <a href="https://support.github.com/contact/feedback" rel="noopener nofollow ugc">official feedback form</a>!</p>
<p>Hope this helps!</p>

**Comment by MarceloHoffmeister** [comment]
<p>I made that! Thank you :grin:</p>

**Comment by Gisgar3** [comment]
<p>Awesome :grinning_face_with_smiling_eyes: glad to have helped!</p>

**Comment by jsoref** [comment]
<p>I couldn’t find your post. Could you link to it?</p>
<p>For the record, because I didn’t find it, I made one…</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github/feedback/discussions/15874">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github/feedback/discussions/15874" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/2119212/181077587-f2b672f2-d1f6-4b6b-bf5b-2bd3ae98fa43.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/2/52c770d0e390f4ca1d639a75059d29dc119f8cf5_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/2/52c770d0e390f4ca1d639a75059d29dc119f8cf5_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/2/52c770d0e390f4ca1d639a75059d29dc119f8cf5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/2/52c770d0e390f4ca1d639a75059d29dc119f8cf5_2_10x10.png"></div>

<h3><a href="https://github.com/github/feedback/discussions/15874" target="_blank" rel="noopener">`Languages` is misleading · Discussion #15874 · github/feedback</a></h3>

  <p>Here's a sample repository: https://github.com/Andrew6rant/Vivy A github.community member posted asking to replace Languages with something else: https://github.community/t/possibly-terminolgy-...</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
